doc/compiler/MemoryManager.md

```diff
@@ -0,0 +1,226 @@
+<!--
+Copyright (c) 2000, 2017 IBM Corp. and others
```

Comment:
Was this file or any of its derivatives created in 2000?

Reply:
Nope, first time file; I'll update it to only have 2017.